pkg/scaffold/api.go

```diff
@@ -108,7 +108,8 @@ func (api *API) buildUniverse() *model.Universe {
 		Plural:     flect.Pluralize(strings.ToLower(api.Resource.Kind)),
 	}
 
-	resourceModel.GoPackage, resourceModel.GroupDomain = util.GetResourceInfo(api.Resource, api.project.Repo, api.project.Domain, api.project.MultiGroup)
+	resourceModel.GoPackage, resourceModel.GroupDomain =
```

Comment:
Would look nicer to split args into multiple lines rather than have the function call on a separate line.

Reply:
Agree 👍